myapp/src/App.js

```diff
@@ -0,0 +1,16 @@
+import LoginPage from "./pages/LoginPage";
+import WelcomePage from "./pages/WelcomePage";
+import { Route, Routes, BrowserRouter } from "react-router-dom";
+
+function App() {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<LoginPage />} />
+        <Route path="home" element={<WelcomePage />} />
+      </Routes>
+    </BrowserRouter>
```

Comment:
Something i feel would be good here, is to separate the routing to a different file. This is fine for now as you only have the 2 routes, but when the routing gets complex and scales it's easy to manage and read if we have separate files for routing. And furthermore sub routing files for individual main routes when expanding further.

For example if the base URL is example.com/ and we have several role based routes like with System2 like /a, /c etc., it helps to have separate files for each of these routes and define the sub routes after these in them.

Reply:
Added role-base routing file and main app routing file.
aa439fed25ecd2380580da968e5ee9fca0a39698